src/main/java/io/confluent/idesidecar/restapi/application/ReflectionConfiguration.java

```diff
@@ -82,7 +91,18 @@
         OAuthBearerUnsecuredValidatorCallbackHandler.class,
         OAuthBearerUnsecuredLoginCallbackHandler.class,
         SaslClientCallbackHandler.class,
-        ScramServerCallbackHandler.class
+        ScramServerCallbackHandler.class,
+        // Schema Registry client classes that are not registered in
+        // https://github.com/quarkusio/quarkus/blob/3.16.3/extensions/schema-registry/confluent/common/deployment/src/main/java/io/quarkus/confluent/registry/common/ConfluentRegistryClientProcessor.java
+        Mode.class,
+        ExtendedSchema.class,
+        Rule.class,
+        RuleKind.class,
+        RuleMode.class,
+        RuleSet.class,
+        SchemaEntity.class,
+        SchemaTags.class,
+        SchemaRegistryServerVersion.class,
```

Comment:
The Quarkus library for Confluent SR does not register all of the classes we will (or might) need at runtime, so we do this here. Most of these are related to data contracts, which were a more recent additional to SR.